.github/workflows/docker-image-release.yml

```diff
@@ -0,0 +1,74 @@
+name: Docker Image (Release)
+
+on:
+  push:
+    tags:
+      - 'pgadmin4-*'
+
+jobs:
+  docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set Up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set Up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      # See https://github.com/docker/metadata-action#readme
+      - name: Docker Meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ${{ secrets.DOCKERHUB_USERNAME }}/pgadmin4
+            ghcr.io/${{ github.repository }}/pgadmin4
+          # generate Docker tags based on the following events/attributes
+          tags: |
+            type=match,pattern=pgadmin4-(.*),group=1
+
+      # See https://github.com/docker/login-action
+      - name: Log In To The DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # See https://github.com/docker/login-action
+      - name: Log In To The GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # See https://github.com/docker/build-push-action#readme
+      # See https://github.com/docker/build-push-action/blob/b1aeb1103e6b3b5648dbd6deaf0559919456ca6b/docs/advanced/tags-labels.md
+      - name: Build And Push
+        id: docker_build
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          platforms: linux/arm64,linux/amd64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6
```

Comment:
Looks like you need to remove

- linux/ppc64le
- linux/s390x
- linux/386
- linux/arm/v7
- linux/arm/v6

Reply:
Fixed. Removed unused OS.

Reply:
![image](https://github.com/theanurin/docker-images/assets/84131104/5ca0bbee-f374-44d2-aa36-e9b40042d328)

Reply:
Fixed.